bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/IFindReplaceLogic.java

```diff
@@ -166,4 +166,15 @@ public interface IFindReplaceLogic {
 	 */
 	public IFindReplaceTarget getTarget();
 
+	/**
+	 * Returns <code>true</code> if searching should be restricted to entire words,
+	 * <code>false</code> if not. This is the case if the respective checkbox is
+	 * turned on, regex is off, and the checkbox is enabled, i.e. the current find
+	 * string is an entire word.
```

Comment:
There are no check boxes at the level of the business logic.
```suggestion
	 * <code>false</code> if not. Searching for whole words requires the given find string to be an entire word and the regex search option to be disabled.
```

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/a0b224321663e63aa99f60da4a2527f993d68f7d